mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/table/MessageIdToImapUid.java

```diff
@@ -29,6 +29,8 @@
 
     String MOD_SEQ = "modSeq";
 
-    String[] FIELDS = { MESSAGE_ID, MAILBOX_ID, IMAP_UID, MOD_SEQ,
+    String THREAD_ID = "threadId";
```

Comment:
Maybe I would have added that constant in `CassandraMessageIds` and then import it here and in CassandraMessageIdTable instead

Reply:
We already defined threadId in AppendResult(separately from ComposedMessageId) and MessageMetaData class. IMO it's not suitable if we put threadId into ComposedMessageId or CassandraMessageIds constrants also.